doc/_static/mpl.css

```diff
@@ -99,3 +99,170 @@ table.property-table td {
     display: inline-block;
     margin: 0 0.5em;
 }
+/* Removes title from toc in dropdwon */
+
+.sd-dropdown>.sd-card-body>.toctree-wrapper>ul>.toctree-l1>a{
+    display: none
+}
+
+.sd-dropdown>.sd-card-body>.toctree-wrapper>ul>.toctree-l1>ul{
+    list-style:none;
+    padding-inline-start:0em;
+}
+
+/* styles dropdown like toctree-toggle */
+details.sd-dropdown{
+    box-shadow: none !important;
+}
+
+details.sd-dropdown summary.sd-card-header,
+details.sd-dropdown summary.sd-card-header+div.sd-summary-content {
+    --pst-sd-dropdown-color: none;
+}
+
+details.sd-dropdown summary {
+    /* padding: 1em; */
+    padding: 0;
+}
+
+.sd-dropdown>.sd-card-body{
+    padding: 0;
+}
+
+:root {
```

Comment:
The rest of the file seems to have been duplicated from above.

Reply:
If you expect to use dropdowns elsewhere (or we already have some), then it would be best to add another class on these ones (an ID must be unique, so you'd have to add a bunch here.)

Reply:
It seems simpler to me to just add a single class to every dropdown here and to the CSS selector rather than come up with unique IDs to use for every dropdown and repeat the CSS selector for each one.